test/integration/plugins/plugins_spec.rb

```diff
@@ -1,7 +1,17 @@
 describe command('grafana-server -v') do
-  its(:stdout) { should match /Version 3.0.3/ }
+  its(:stdout) { should include 'Version 6.2.0' }
```

Comment:
Should we make this more generic, such as just look for 'Version' as otherwise it adds maintenance on us when upstream release anything?